site-manage/site-manage-tool/tool/src/webapp/vm/sitesetup/chef_site-siteInfo-manageOverview.vm

```diff
@@ -3,10 +3,10 @@
 
         <input type="hidden" name="eventSubmit_doManage_overview_option" value="x" />
         <input type="hidden" name="option" value="cancel" />
-        <div class="page-header">
-            <h1>$tlang.getString("manover.header") <span class="highlight">$!site.Title</span></h1>
-        </div>
-        #if ($alertMessage)<div class="alertMessage">$tlang.getString("manover.alert") $validator.escapeHtml($alertMessage)</div>#end
+        <h3>
```

Comment:
Another case of the page heading being reverted...

Reply:
Is this better? I think a reason it was being reverted was that in master, there's places where there's conflicts (like some alertmessage areas) which affected what ended up being pushed

Reply:
No, it is still showing the removal of the .page-header div and the change back to an h3 here in GitHub.

Reply:
Oh, sorry: it took me to the old version. Let me do another review...